keps/sig-network/2086-service-internal-traffic-policy/README.md

```diff
@@ -228,7 +249,301 @@ Metrics representing Services being black-holed will be added. This metric can i
 
 * **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
 
-No, but this will be manually tested prior to beta. Automated testing will be done if the test tooling is available.
+#### API Enablement Rollback
```

Comment:
Sorry can't comment on unchanged lines so commenting here:

L248: Have those metrics been added? Can you please update this answer?
[I think it's sync_proxy_rules_no_local_endpoints_total - if so, please just update the answer]

Reply:
Thanks, updated